packages/generator-electrode/generators/app/index.js

```diff
@@ -57,6 +56,16 @@ module.exports = generators.Base.extend({
       required: false,
       desc: 'Content to insert in the README.md file'
     });
+    //Flag to check if the OSS generator is being called as a subgenerator
+    if (this.options && this.options.isExtended) {
+      this.isExtended = true;
+    }
+    if (this.options && this.options.serverType) {
+      this.serverType = this.options.serverType;
+    }
+    if (this.options && this.options.githubUrl) {
+      this.githubUrl = this.options.githubUrl;
```

Comment:
why do you have to check each one?  Why not just directly assign each?  You are checking them every where anyways.